### PR TITLE
Fixing - IDENTITY-3933

### PR DESCRIPTION
**Pull request by JKAUSHALYA**
https://wso2.org/jira/browse/IDENTITY-3933

### CODE REVIEW

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 1]
```diff
@@ -182,48 +180,43 @@ public IdentityProvider getResidentIdP(String tenantDomain)
...
+            oauth2TokenEPUrl = IdentityUtil.getServerURL("oauth2/token", false);
         }
         if(StringUtils.isBlank(oauth2UserInfoEPUrl)){
-            serverUrl = IdentityUtil.getServerURL("", false) + "/";
-            oauth2UserInfoEPUrl = serverUrl + "oauth2/userinfo";
+            oauth2UserInfoEPUrl = IdentityUtil.getServerURL("oauth2/userinfo", false);
         }
         if(StringUtils.isBlank(passiveStsUrl)){
-            passiveStsUrl = serverUrl + "passivests";
+            passiveStsUrl = IdentityUtil.getServerURL("passivests", false);
```
Comment (darshanasbg) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: should be,
passiveStsUrl = IdentityUtil.getServerURL("passivests", true);

**Comment by darshanasbg** [comment]
Reviewed..!

File: components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java [hunk 2]
```diff
@@ -182,48 +180,43 @@ public IdentityProvider getResidentIdP(String tenantDomain)
...
-            oauth1RequestTokenUrl = serverUrl + "oauth/request-token";
+            oauth1RequestTokenUrl = IdentityUtil.getServerURL("oauth/request-token", true);
         }
         if(StringUtils.isBlank(oauth1AuthorizeUrl)){
-            oauth1AuthorizeUrl = serverUrl + "oauth/authorize-url";
+            oauth1AuthorizeUrl = IdentityUtil.getServerURL("oauth/authorize-url", true);
         }
         if(StringUtils.isBlank(oauth1AccessTokenUrl)){
-            oauth1AccessTokenUrl = serverUrl + "oauth/access-token";
+            oauth1AccessTokenUrl = IdentityUtil.getServerURL("oauth/access-token", true);
```
Comment (lalaji) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: can we keep these inline string values as constants [eg:"oauth2"].I noticed some of those are using from another code blocks as well.

Comment (JKAUSHALYA) [inline components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java]: Fixed in https://github.com/wso2/carbon-identity/pull/1701

**Pull request merged**